src/client/containers/PullRequestSummary/ReviewSection/__tests__/index.js

```diff
@@ -0,0 +1,46 @@
+import chai from 'chai'
+import {
+  getReviews,
+} from '../index'
+
+const expect = chai.expect
+
+describe('getReviews', () => {
+  it('full data', () => {
+    const user1 = 'user1'
+    const user2 = 'user2'
+    const users = {
+      1: user1,
+      2: user2,
+    }
+    const review1 = {
+      status: 'ChangesetStatus.UNDER_REVIEW',
+      user: 1,
+    }
+    const review2 = {
+      status: 'ChangesetStatus.APPROVED',
+      user: 2,
+    }
+    const expectedData = [
+      { ...review1, user: user1 },
+      { ...review2, user: user2 },
+    ]
+    const pullRequests = {
+      12: {
+        reviews: [review1, review2],
+        otherParameter: 'otherParameter',
```

Comment:
Question here: are reviews entities? Because if yes, then I think they will need their own selector in `ducks/pullrequests/selectors` and both the test and the getReviews selector will have to be rewritten a little.